sdk/src/types/globals.ts

```diff
@@ -0,0 +1,18 @@
+// using "| any" to ensure compatibility with various "Fetch" alternative function signatures
+export type FetchInterface = (input: string | any, init?: RequestInit | any) => Promise<unknown>;
+
+export type UrlInterface = typeof URL;
+
+export type WebSocketConstructor = {
+	new (url: URL, protocols?: string | string[]): WebSocketInterface;
+};
+
+export interface WebSocketInterface {
```

Comment:
`FetchInterface` and `UrlInterface` seem to be `type`s whereas only `WebSocketInterface` is an `interface` - We might wanna stick to one thing for consistency? ( super nitpicky I know :pray: )